src/qt/sendcoinsdialog.cpp

```diff
@@ -379,20 +376,25 @@ void SendCoinsDialog::on_sendButton_clicked()
     if (!PrepareSendText(question_string, informative_text, detailed_text)) return;
     assert(m_current_transaction);
 
-    const QString confirmation = model->wallet().privateKeysDisabled() ? tr("Confirm transaction proposal") : tr("Confirm send coins");
-    const QString confirmButtonText = model->wallet().privateKeysDisabled() ? tr("Create Unsigned") : tr("Send");
-    SendConfirmationDialog confirmationDialog(confirmation, question_string, informative_text, detailed_text, SEND_CONFIRM_DELAY, confirmButtonText, this);
+    const QString confirmation = tr("Confirm send coins");
+    const QString confirmButtonText = tr("Send");
+    const QString psbt_button_text = tr("Create Unsigned");
+
+    // Show the Create Unsigned button if set in the options or if private keys are disabled.
+    bool show_unsigned_button = model->getOptionsModel()->getMakeUnsignedButton() || model->wallet().privateKeysDisabled();
```

Comment:
Can we move this up to have the `||` only done once?

Reply:
It is though.